bot/utils/scheduling.py

```diff
@@ -157,18 +157,20 @@ def _task_done_callback(self, task_id: t.Hashable, done_task: asyncio.Task) -> N
                 self._log.error(f"Error in task #{task_id} {id(done_task)}!", exc_info=exception)
 
 
-def create_task(*args, **kwargs) -> asyncio.Task:
+def create_task(suppress_exceptions: t.Tuple[Exception], *args, **kwargs) -> asyncio.Task:
```

Comment:
Making this the first argument and non-optional breaks all other uses of this function. The coroutine should be the first argument and the function should not require callers to always specify exceptions. Consider using `*args` for the exceptions since it'd be more convenient for callers.

Reply:
> Making this the first argument and non-optional breaks all other uses of this function. The coroutine should be the first argument and the function should not require callers to always specify exceptions. Consider using `*args` for the exceptions since it'd be more convenient for callers.

But it's already using `*args`. How about making it a kwarg?

Reply:
It's not actually using it. It was just done this way to make it future-proof against changes to `asyncio.create_task`'s signature, but this is unlikely to happen anyway.